### ISSUE TITLE
semi-enzymatic PSMs

### ISSUE THREAD

**Issue by tobiasko** [Nesvilab/IonQuant#5]
Dear IonQuant developers,

I wanted to ask if you meanwhile checked the amount of semi-enzymatic PSMs on a different dataset than PXD010012 (your preprint manuscript, section Semi-tryptic Peptide Monitoring)? I just had a look at results comparing semi tryptic and fully tryptic searches on the triple hybride proteome data (PXD014777). For a single replicate (A_4) I get

```
tobiasko@fgcz-r-033:/scratch/FRAGPIPE/WU241365/A_4$ wc -l p*.tsv
     48365 peptide.tsv
      8588 protein.tsv
     94138 psm.tsv
    151091 total
```
in a tryptic search vs.

```
tobiasko@fgcz-r-033:/scratch/FRAGPIPE/WU241718/A_4$ wc -l p*.tsv
   57437 peptide.tsv
    8781 protein.tsv
  107650 psm.tsv
  173868 total
```
for a semi tryptic.

That is a 18% increase in contrast to the +63% for PXD010012. This would support your comment that "the high rates of semi-enzymatic PSMs may be specific to the timsTOF dataset used in this work".

What also puzzles me: Given that the majority of tryptic ions should be +2 one would expect to generate +1 product ions by gas-phase fragmentation events. But these should not be recorded, since +1 ions are excluded by PASEF methods using the polygon selection. Have you ever looked at the z state distribution of these semi tryptic ions and their likely parent ions?

I have never recorded PASEF data with and without polygon selection, but it might be important to understand to what extent gas-phase fragmentation actually occurs on a normally tuned timsTOF Pro. Maybe this +1 cloud outside of the polygon is not only chemical noise but products coming from the trapping process? Your finding that longer accumulation times correlated with increased semi-specific ions would also support this.

Best,
Tobi

**Comment by fcyu** [comment]
Hi Tobi,

Thanks for this insightful finding. We also analyzed the data from PXD014777. Both of the three-organism data set and the three HeLa QC data set got a similar gain (~17%) from semi-tryptic searching. We've added some descriptions to the revised manuscript.

Your follow-up questions are also very good and interesting. Sarah and I actually had a very brief discussion regarding this but there was no conclusion (I honestly cannot remember what was happening at that time). Maybe it's time for us to revisit this one. Will keep you updated when we have something come up.

Best,

Fengchao
